composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "krowinski/php-mysql-replication",
+  "name": "dominiquegerber/php-mysql-replication",
```

Comment:
😭

Reply:
Oh crap... Please don't take offense, I don't mean to steal your work ! I renamed the package to avoid any confusion to use it in my own project until a fix for the negative number issue is available in yours. I'll sort that out.